_posts/2020-07-21-why-paper-and-pencil-is-my-best-tool.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title:  "Why paper and pencil are my best tools"
+date:   2020-07-21 23:30:41 -0300
+categories: tools
+---
+Try to think about how much time we spend looking for an ideal tool to taking our notes, reminders, scheduling, sketching… A tool that give us freedom and flexibility to do everthing we need. Yeah .. so try to look around. Can you see this tool? There it is! A simple paper and pencil right next to you.
+
+I can’t imagine how much ideas and time I saved using this and I will try explain why this works so well to me.
+
+All my programmer life I’ve tried to use many tools that improve my capabilities to remember things, organize TODO lists, improve creativity and etc. One thing that works so well on all my needs, with freedom and combined all other tools that have separated and specialized functions, is the paper and pencil.
+
+> Ok, Guilherme. But what about smartphone screen notifications for your schedule todo list/meeting?
+
+Ok. I know and I use this too as an extra (Yeah.. as an extra, not as the main tool)
+
+Organizing into paper or a notebook turns me more disciplined ever, just because I can create mindsets, draw ever I need to explore more and when I must remember some topics with emfhasis. This kind of things enforces our memory so well that tecnologies will never do the same.
```

Comment:
Typos: "Draw ever", "enfhasis".

Reply:
I didn't understand. Google translate tells me that is ok with these words... Is it really wrong?